### PR TITLE
fix: Fix license search by alternative names and remove code duplicates

### PR DESCRIPTION
**Pull request by o-kopysov**
# Pull Request

## Description

Current PR contains:
- fix for the license search by alternative names
- extension of the test DB dump with license alternative names
- removed code duplicates according to the SAM tool check

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [ ] Test Coverage update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

**Test Configuration**:
* Java: v17
* LPVS Release: v1.5.0

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] My code meets the required code coverage for lines (90% and above)
- [X] My code meets the required code coverage for branches (80% and above)
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by tdrozdovsky — APPROVED** [review]

File: src/main/java/com/lpvs/util/LPVSWebhookUtil.java [hunk 1]
```diff
@@ -109,13 +109,12 @@ public static boolean checkPayload(String payload) {
...
-     * @return The organization name.
-     * @throws IllegalArgumentException If the provided LPVSQueue object is null or if the repository URL is absent.
+     * @param webhookConfig the LPVSQueue object to check
+     * @throws IllegalArgumentException If the webhook configuration is null or if the repository URL is absent.
      */
-    public static String getRepositoryOrganization(LPVSQueue webhookConfig) {
+    private static void checkWebhookConfig(LPVSQueue webhookConfig) {
         if (null == webhookConfig) {
             log.error("Webhook Config is absent");
             throw new IllegalArgumentException("Webhook is absent");
```
Comment (t-naumenko) [inline src/main/java/com/lpvs/util/LPVSWebhookUtil.java:120]: I think it is better to keep the same text of the error in log.error and in the IllegalArgumentException: "Webhook Config is absent"

**Review by t-naumenko — APPROVED** [review]
There is a tiny comment, but in general it looks good to me.
Approved

**Review by o-konoval — APPROVED** [review]
Good job. It will increase quality of detection.
Approved.

**Review by tiokim — APPROVED** [review]
LGTM

**Pull request merged**